doc/compressed_transactions.md

```diff
@@ -0,0 +1,49 @@
+# Compressed Transaction Schema
+
+## 1. Abstraction
+With this transaction compression schema we reduce the size of all transactions by at least 15-20% and up to 40% with signature recovery. The main current uses for this will be steganography or other low bandwidth channels for passing transaction data.
+
+## 2. Methods
+The four main methods to achieve a lower transactions size are:
+1. Adding a few bytes of data before the transaction and each of its inputs and outputs to determine the structure of the following data.
+2. Removing all excess white space and fillers with the use of Variable Integers.
```

Comment:
In 61504d1a4d7ea545b1880686e426b1acae7d26f0:

"whitespace" -> "frequently unused bits in the version, locktime, and sequence fields"